openai/src/commonMain/kotlin/com/xebia/functional/xef/conversation/llm/openai/AutoClose.kt

```diff
@@ -0,0 +1,14 @@
+package com.xebia.functional.xef.conversation.llm.openai
+
+import com.aallam.openai.client.Closeable
+import com.xebia.functional.xef.conversation.AutoClose
+
+/** integration to aallam's [Closeable] */
+internal fun <A : Closeable> AutoClose.autoClose(closeable: A): A {
```

Comment:
I thought we already had AutoClose declared elsewhere unless we got rid of it.

Reply:
The definition of the `AutoClose` class is inside the core module. Because this is a bridge to aallam's custom AutoClose, I put this into the openai module as it does not belong into the core package. Equivaelently we provide a bridge to Ktor's AutoClose, which you might be referrring to. Does that answer your concern?